client.go

```diff
@@ -589,6 +615,42 @@ func New(cfg Config) (c Client, err error) {
 		).WithOpenTracing(), nil),
 	}
 
+	c.Identity = &Identity{
+		Acp: identity.New(httptransport.NewWithClient(
+			cfg.IssuerURL.Host,
+			c.apiPathPrefix(cfg.VanityDomainType, "/api/identity/%s"),
+			[]string{cfg.IssuerURL.Scheme},
+			client,
+		).WithOpenTracing(), nil),
+	}
+
+	c.IdentitySelf = &IdentitySelf{
+		Acp: identityself.New(httptransport.NewWithClient(
+			cfg.IssuerURL.Host,
+			c.apiPathPrefix(cfg.VanityDomainType, "/%s/%s"),
+			[]string{cfg.IssuerURL.Scheme},
+			client,
+		).WithOpenTracing(), nil),
+	}
+
+	c.IdentityRoot = &IdentityRoot{
+		Acp: identityroot.New(httptransport.NewWithClient(
+			cfg.IssuerURL.Host,
+			c.apiPathPrefix(cfg.VanityDomainType, "/api/identity/%s"),
```

Comment:
This format string should also be reviewed. From `acp/internal/identity/handler/rest/root/api_server.go`:
```
var RoutingPathPrefixes = kernel.NewSubrouterPathPrefix(
    "/api/identity/system",
    "/api/identity/system",
    "/api/identity/system",
)
```
which would mean that there is no {tid}, even when the domain is not a vanity domain.